### DISCUSSION TITLE
Has been Logscan definetly removed fom Security Onion plugins?

### DISCUSSION THREAD

**Discussion by mmartinrami** [Security-Onion-Solutions/securityonion#13060] [Unsupported Versions] [answered]
In 2.3 Security Onion  version a logscan beta was available but in  the latest version 2.4 was removed. So why?  an will it be available in the future?

**Comment by dougburks** [comment, marked as answer]
Logscan was only provided as a BETA feature in 2.3 and never reached full release. 

We are looking to introduce a more integrated ML tool going forward, but there's no ETA yet.
